### ISSUE TITLE
Why is the U extension "privileged"? This is probably a typo.

### ISSUE THREAD

**Issue by james-ball-qualcomm** [riscv-software-src/riscv-unified-db#287]
![image](https://github.com/user-attachments/assets/4d0d4cc1-2523-4f5b-9cc0-79d48e611dbd)

**Comment by dhower-qc** [comment]
Not a typo: The U extension introduces U-mode and the CSRs to control it (from M/S mode).

**Comment by james-ball-qualcomm** [comment]
Glad I asked.

**Comment by james-ball-qualcomm** [comment]
What exactly does privileged mean in the schema for a CSR? I'm re-opening this as a documentation request.

From https://github.com/riscv-software-src/riscv-unified-db/blob/main/arch/README.adoc


![image](https://github.com/user-attachments/assets/4375fe50-84fe-4464-8f84-b5f467d36948)

**Comment by dhower-qc** [comment]
My understanding that "privileged" shows up in the priv manual and "unprivileged" shows up in unpriv. That said, those fields are not used anywhere. I only added them because RVI processes seem to make a distinction (though with the combined ISA manual, maybe it really doesn't matter anymore).

**Comment by james-ball-qualcomm** [comment]
This raises a good point. I don't see a "U" extension defined in either the Priv or Unpriv ISA manual. So, is the "U" extension actually something we've invented in the UDB? I think we should add some way to mark extensions like Sm and Smpmp that we've created in UDB.

Here's the best I can find about User-mode. 

![image](https://github.com/user-attachments/assets/67412118-4869-404b-b8d8-84c50c0edd44)

**Comment by dhower-qc** [comment]
Yes, that's the only place I know where it is mentioned. So, the intent seems to be that U is an extension, but exactly what that means has to be inferred.
